users/class_tests/users_tests.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from phonenumber_field.phonenumber import PhoneNumber
+
+from users.models import User
+
+
+NAME = "user"
+EMAIL = "user@gmail.com"
+PASSWORD = "AdminU$er123"
+PHONE_NUM = "+972544651892"
+
+MISSING_REGION = "Missing or invalid default region."
+NOT_VALID_PHONE = "The phone number entered is not valid."
+NOT_INTERPRETED = "Could not interpret numbers after plus-sign."
+NOT_PHONE_NUMBER = "The string supplied did not seem to be a phone number."
+NAME_LENGTH_ERROR = "Ensure this value has at most 30 characters."
+EMAIL_ERROR = "Email is not valid"
+
+
+@pytest.fixture
+def user0():
+    return User(email=EMAIL, phone_number=PHONE_NUM, password=PASSWORD, username=NAME)
+
+
+@pytest.fixture
+def persist_user(db, user0):
+    user0.save()
+    return user0
+
+
+@pytest.mark.django_db()
```

Comment:
done. thanks 😄 
** the function user0 is outside the class because a lot of other tests are using it 
and the including of this function in the class has only cause errors
so I have decided to exclude it from the main test class